src/GitLink/project.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "dependencies": {
+    "Catel.Core": "4.3.0",
+    "GitTools.Core": "1.0.0-unstable0021",
+    "ImpromptuInterface": "6.2.2",
+    "LibGit2Sharp": "0.21.0.176",
+    "ReadOnlySourceTree": {
+      "version": "0.1.37-beta",
+      "suppressParent": "none"
+    },
+    "Nerdbank.GitVersioning": {
```

Comment:
1. automatic generation of `AssemblyVersionAttribute` with a major.minor version.
1. automatic generation of `AssemblyFileVersionAttribute` with a major.minor.githeight.commitid version.
1. automatic generation of `AssemblyInformationalVersion` with major.minor.githeight+gCommitId
1. nuget package builds with major.minor.height version. and include a -gCommitId prerelease tag when not built from the "public release" branch/tag naming patterns.
1. generated nuspec uses commit id in place of a `$(GitCommitId)` macro so that license and icon links never break and always point at the original license and icon for that package version.

And all this in all local builds and cloud builds, exactly the same, for reproducibility.